src/params.rs

```diff
@@ -259,6 +265,40 @@ impl<S: AsRef<str> + std::fmt::Debug + PartialEq<S> + From<&'static str>> TryFro
 	}
 }
 
+/// Enforcement strictness for file sandbox
```

Comment:
This still requires the information, that you can't create a new VM in the same process if enforcement is active.